src/Take.Blip.Builder/Utils/SalesForce/Models/AuthorizationResponse.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+
+namespace Take.Blip.Builder.Utils.SalesForce.Models
+{
+    /// <summary>
+    /// Sales force authorization response
+    /// </summary>
+    public class AuthorizationResponse
+    {
+        /// <summary>
+        /// Access token
+        /// </summary>
+        [JsonProperty("access_token")]
```

Comment:
I assume this is snake case because it's interacting with an external API, correct?

Reply:
Correct